src/Microsoft.Data.SqlClient/tests/FunctionalTests/AlwaysEncryptedTests/SqlColumnEncryptionCspProviderShould.cs

```diff
@@ -97,7 +98,7 @@ public class InvalidDecryptionParameters : DataAttribute
             private const string TCE_InvalidCiphertextLengthInEncryptedCEK = "The specified encrypted column encryption key's ciphertext length: 128 does not match the ciphertext length: 256 when using column master key (asymmetric key) in 'Microsoft Enhanced RSA and AES Cryptographic Provider/KeyName'. The encrypted column encryption key may be corrupt, or the specified Microsoft Cryptographic Service provider (CSP) path may be incorrect.\r\nParameter name: encryptedColumnEncryptionKey";
             private const string TCE_InvalidSignatureInEncryptedCEK = "The specified encrypted column encryption key's signature length: 128 does not match the signature length: 256 when using column master key (asymmetric key) in 'Microsoft Enhanced RSA and AES Cryptographic Provider/KeyName'. The encrypted column encryption key may be corrupt, or the specified Microsoft cryptographic service provider (CSP) path may be incorrect.\r\nParameter name: encryptedColumnEncryptionKey";
             private const string TCE_InvalidSignature = "The specified encrypted column encryption key signature does not match the signature computed with the column master key (asymmetric key) in 'Microsoft Enhanced RSA and AES Cryptographic Provider/KeyName'. The encrypted column encryption key may be corrupt, or the specified path may be incorrect.\r\nParameter name: encryptedColumnEncryptionKey";
-
+            private string       TCE_InvalidCspKeyId = $"Internal error. Invalid key identifier: 'KeyName/{DUMMY_KEY}'. Verify that the key identifier in column master key path: 'Microsoft Enhanced RSA and AES Cryptographic Provider/KeyName/{DUMMY_KEY}' is valid and exists in the CSP.\r\nParameter name: masterKeyPath";
```

Comment:
Nit: plz format

Reply:
Oh do you mean the space between string and TCE_InvalidCspKeyId? I did that on purpose to make them look nicer. I can remove the space if you want.